STYLEGUIDE.md

```diff
@@ -188,106 +200,112 @@ For multiple lines of code or an example command, use a
 [code block](#code-block).
 
 Do not use in-line formatting for anything that should be in 
-[bold text](#bold-text)
+[bold text](#bold-text).
 
 ### Kong
-Use proper case
+Use proper case.
 
 ### Kong Admin
-Use Admin instead
+Use [**Admin** or **admin**](#admin-admin) instead.
```

Comment:
This link was confusing, would it be better to say "Use **Admin** or **admin** (See #admin-admin)

Reply:
How's this?

Use **Admin** or **admin** (see [Admin, admin](#admin-admin)) instead.

```suggestion
Use **Admin** or **admin** (see [Admin, admin](#admin-admin)) instead.
```